src/js/menu/menu-button.js

```diff
@@ -88,8 +88,11 @@ class MenuButton extends Component {
 
     if (this.items && this.items.length <= this.hideThreshold_) {
       this.hide();
+      this.menu.contentEl_.removeAttribute('role', 'menu');
```

Comment:
```suggestion
      this.menu.contentEl_.removeAttribute('role');
```
`removeAttribute()` takes one argument.